### PR TITLE
Create project functional inputs

### PR DESCRIPTION
**Pull request by Andresmagic**
# What does this PR do?

Migrate the simple inputs to create a project

# Where should the reviewer start?

You can start checking ProjectForm.tsx and review every input in there 

# How should this be manually tested?

1. From the landing page click the button of 'New Proposal'
2. Fill the inputs 
3. Click the 'Create Project' button 
4. Check in your terminal your inputs from the console.log

# Any background context you want to provide?

There are still some inputs that are needed to create a project that are commentated.
This PR doesn't include the useFetcher for creating the project, just the base inputs.

# What are the relevant tickets?

https://github.com/wizeline/remix-project-lab/pull/48

# Screenshots
image
image
image

This PR was made with the collaboration of: @dmojarrot 
# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

**Comment by dmojarrot** [comment]
Text editor input now working.

Screen Shot 2022-10-17 at 14 05 21

Screen Shot 2022-10-17 at 14 05 45

File: app/core/components/DisciplineSelect/index.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+interface DisciplinesSelectProps {
+  defaultValue?: any[];
+  customOnChange?: (arg: any) => void;
+  fullWidth?: boolean;
+  name: string;
+  label: string;
+  helperText?: string;
+  outerProps?: PropsWithoutRef<JSX.IntrinsicElements["div"]>;
+  size?: "small" | "medium" | undefined;
```
Comment (martinrobled0) [inline app/core/components/DisciplineSelect/index.tsx:14]: creo que ahi esta de mas el undefined

**Review by martinrobled0 — CHANGES_REQUESTED** [review]
just the undefined things but LGTM

File: app/core/components/DisciplineSelect/index.tsx (hunk 1 above)
Comment (Andresmagic) [reply app/core/components/DisciplineSelect/index.tsx:14]: Cierto!, igual ese input aún está comentado en el ProjectForm y no es parte de los que están funcionales, pero igual lo cambio de una vez:)

File: app/core/components/ProjectOwnerField/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  helperText?: string;
+}
+
+export const ProjectOwnerField = ({
+  name,
+  label,
+  owner,
+  helperText,
+}: ProfilesSelectProps) => {
+  const profiles: readonly any[] = ["diego", "jorge", "jose"];
```
Comment (juanan0309) [inline app/core/components/ProjectOwnerField/index.tsx:17]: We shouldn't have this hardcoded, we have a `Profiles` table that help us with this info

File: app/core/components/RelatedProjectsSelect/index.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  fullWidth,
+  name,
+  label,
+  helperText,
+  outerProps,
+  size,
+  style,
+  thisProject,
+}: RelatedProjectsSelectProps) => {
+  const projects = ["Project 1", "Project 2", "Project 3", "Project 4"];
```
Comment (juanan0309) [inline app/core/components/RelatedProjectsSelect/index.tsx:31]: We shouldn't have this hardcoded

File: app/core/components/SkillsSelect/index.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  defaultValue = [],
+  fullWidth,
+  name,
+  label,
+  helperText,
+  outerProps,
+  size,
+  style,
+}: SkillsSelectProps) => {
+  const skills = ["React", "Node", "Python", "Java", "C++", "C#"];
```
Comment (juanan0309) [inline app/core/components/SkillsSelect/index.tsx:29]: We shouldn't have this hardcoced

File: app/models/project.server.ts [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import type { Profiles, Projects } from "@prisma/client";
 import { Prisma } from "@prisma/client"
 
+
```
Comment (juanan0309) [inline app/models/project.server.ts:4]: Is this blank space necessary?

File: app/routes/projects/components/ProjectForm.tsx [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+    setHelpWanted(!helpWanted);
+  };
+
+  const statuses = [
+    { name: "Active" },
+    { name: "Inactive" },
+    { name: "Completed" },
+  ];
+
+  const tiers = [{ name: "0" }, { name: "1" }, { name: "2" }, { name: "3" }];
```
Comment (juanan0309) [inline app/routes/projects/components/ProjectForm.tsx:42]: I think this shouldn't be hardcoded

**Review by juanan0309 — CHANGES_REQUESTED** [review]

File: app/root.tsx [hunk 7]
```diff
@@ -41,13 +45,48 @@ type LoaderData = {
...
+                    slackChannel,
+                    skills,
+                    labels,
+                    relatedProjects,
+                    innovationTiers,
+                    projectMembers})*/ 
+  return redirect("/projects");
+}
+
+
```
[the remark this answers is not in the document]
Comment (Andresmagic) [reply app/root.tsx:87]: I tried doing this in the create project file but I got errors that say that it should be done in the `root.tsx` . But this was just to see the console log and seeing if the formData was working, but in the end i think we're going to delete this from the root and do all with the useFetcher

File: app/models/project.server.ts (hunk 5 above)
Comment (Andresmagic) [reply app/models/project.server.ts:4]: No, ups

**Pull request closed without merging**